specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2021-10/schemaregistry.json

```diff
@@ -0,0 +1,497 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Schema Registry",
+    "version": "2021-10",
+    "description": "Azure Schema Registry is as a central schema repository, with support for versioning, management, compatibility checking, and RBAC."
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": true,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/$schemaGroups": {
+      "get": {
+        "summary": "Get list of schema groups.",
+        "description": "Gets the list of schema groups user is authorized to access.",
+        "x-ms-examples": {
+          "Get schema by ID": {
+            "$ref": "./examples/OperationSchema_GetSchemaGroups.json"
+          }
+        },
+        "operationId": "SchemaGroups_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SchemaGroups"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Schema Registry service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/$schemaGroups/$schemas/{id}": {
+      "get": {
+        "summary": "Get a registered schema by its unique ID reference.",
+        "description": "Gets a registered schema by its unique ID.  Azure Schema Registry guarantees that ID is unique within a namespace. Operation response type is based on serialization of schema requested.",
+        "x-ms-examples": {
+          "Get schema by ID": {
+            "$ref": "./examples/OperationSchema_GetById.json"
+          }
+        },
+        "operationId": "Schema_GetById",
+        "produces": [
```

Comment:
I think this should only have the last content-type, right?

Reply:
Correct, will remove in next iteration

Reply:
I can remove `text/plain` but removing `application/json` gets tricky. Removing it from this list is semantically correct, but doing so causes the example validation to fail because the returned schema example in `OperationSchema_GetByID` looks structurally familiar to json, but is really an avro-serialized json schema.

Reply:
I believe @lmazuel should be able to help get around the validation errors. I think we want to make sure this is merged in before GAing the libraries.

Reply:
@nickghardwick the Swagger needs to represent the truth, and there is no scenario that I'm aware that where `application/json` is an actual real value, so it shouldn't be in the Swagger.
If that makes validation fials, two things can happen. 1) I can force merge the PR with a justification of validation bug 2) start an email with the CI team to get the bug fixed (asynchronously of your PR getting merged)

Reply:
@nickghardwick  Looking at the error:
```
operationId: Schema_GetById
scenario: Get schema by ID
source: response
responseCode: '200'
severity: 1
code: INVALID_CONTENT_TYPE
details:
  code: INVALID_CONTENT_TYPE
  message: >-
    Invalid Content-Type (application/json).  These are supported:
    application/json; serialization=avro
  path: ''
  jsonPath: ''
```

And seing the Swagger here defines correctly "produces: application/json; serialization=avro", and the x-ms-example defines correctly `"Content-Type": "application/json;serialization=avro"` feels indeed a bug to me.

Reply:
https://github.com/Azure/oav/issues/699